spp_farmer_registry_base/__manifest__.py

```diff
@@ -19,6 +19,7 @@
         "g2p_registry_membership",
         "spp_base_gis",
         "spp_land_record",
+        "spp_registrant_import",
```

Comment:
@emjay0921 Question: do we really need to add the module `spp_registrant_import` in the manifest of `spp_farmer_registry_base`?

I think you can fix the issue without adding the module in the manifest

Reply:
Yeah I forgot to remove that :)

Reply:
Done @reichie020212